code/game/gamemodes/events/hivemind_invasion.dm

```diff
@@ -8,7 +8,6 @@
 /datum/storyevent/hivemind
 	id = "hivemind"
 	name = "Hivemind Invasion"
-	req_crew = 16
```

Comment:
Hivemind on lowpop is very very bad, don't do that.

Reply:
~~skill issue~~

woops, removed one line too many, gimme a bit

Reply:
there, lowered it to 14 pop